build.sbt

```diff
@@ -2,22 +2,23 @@ name := "Akka SMPP"
 
 organization := "akkasmpp"
 
-version := "0.3.0-SNAPSHOT"
+version := "0.4.0-SNAPSHOT"
 
-scalaVersion := "2.11.8"
+scalaVersion := "2.12.1"
 
-resolvers += "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
+crossScalaVersions := Seq("2.11.9", "2.12.1")
 
+resolvers += "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
```

Comment:
Is this actually needed anymore? It might be leftover from when I was pulling down RCs or something